pyiron_atomistics/calphy/job.py

```diff
@@ -57,10 +67,19 @@
         "liquid_fraction": 0.05,
         "pressure": 0.5,
     },
+    "nose_hoover": {
+        "thermostat_damping": 0.1,
+        "barostat_damping": 0.1,
+    },
+    "berendsen": {
+        "thermostat_damping": 100.0,
+        "barostat_damping": 100.0,
+    },
+    "calc": {},
```

Comment:
Here, you mentioned you do not need this?

Reply:
Done